internal/helm/helm.go

```diff
@@ -96,8 +96,10 @@ func RemoveService(logger logr.Logger, cluster *kubernetes.Cluster, app models.A
 		return errors.Wrap(err, "create a helm client")
 	}
 
-	// err == nil => passed through unchanged
-	return errors.Wrap(client.UninstallReleaseByName(names.ServiceReleaseName(app.Name)), "cleaning up release")
+	// Ignore errors. The release may not be present (for example due to an aborted deployment)
```

Comment:
Fine to ignore the error but I would at least log it, to check if something else happened.

The best would be to check if the error is for a not present release but it seems that there are no exported errors to check in the Helm client library, and checking the error string is too brittle. So let's keep it like this and just add a log.

Reply:
I actually got a multitude of different errors, depending where the server was in the creation/abort of the helm release. `release not found` was the simplest. Other's were about some (deletion?) hook, I think. IOW I found ignoring a specific error to be fraught with peril.

Logging is fine, and will do.

Reply:
Done. Commit f5e51745c63f6b23d0de71b04b23b313fd5403a2